pageserver/src/tenant/timeline.rs

```diff
@@ -2906,12 +3000,12 @@ impl Timeline {
         // in-memory layer from the map now.
         {
             let mut layers = self.layers.write().await;
-            let l = layers.frozen_layers.pop_front();
+            let l = layers.0.frozen_layers.pop_front();
```

Comment:
Do we have an entry for `l` in the `LayerMapping`?
If not, should explain here why.

Reply:
`LayerMapping` only stores persistent layers, will add a comment